nh_eobs/tests/nh_clinical_frequencies_ews/test_get_refusal_frequency.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+from openerp.addons.nh_ews.tests.common import clinical_risk_sample_data
+from openerp.tests.common import TransactionCase
+
+
+class TestGetRefusalFrequency(TransactionCase):
+    def setUp(self):
+        super(TestGetRefusalFrequency, self).setUp()
+        self.test_utils = self.env['nh.clinical.test_utils']
+        self.frequencies_model = self.env['nh.clinical.frequencies.ews']
+        self.config_model = self.env['ir.config_parameter']
+
+        self.test_utils.admit_and_place_patient(create_placement=False)
+        self.test_utils.copy_instance_variables(self)
+
+        self.config_model.set_param('placement', 30)
+        self.config_model.set_param('transfer', 60)
+        self.config_model.set_param('obs_restart', 120)
+        self.config_model.set_param('no_risk', 240)
+
+    def call_test(self):
+        refused_obs_activity = \
```

Comment:
Maybe add a docstring here?

Something like: `Refuse the open observation and return the new frequency in order to verify is correct`

Reply:
Yeah sure.